.github/workflows/create-tag.yaml

```
@@ -14,7 +14,7 @@ defaults:
 jobs:
   Create-Tag:
     runs-on: ubuntu-18.04
-    if: github.repository == 'rook/rook' && contains(secrets.RELEASE_ADMINS, github.actor)
+    if: github.repository == 'rook/rook' && contains('travisn,leseb,BlaineEXE' github.actor)
```

Is it possible to test a similar condition on your fork before we merge this? Perhaps on a different action, just to make sure the syntax is correct.
```suggestion
    if: github.repository == 'rook/rook' && contains('travisn,leseb,BlaineEXE', github.actor)
```

---

Let's also make this list the same as maintainers, so please add `jbw976,galexrt,satoru-takeuchi`

---

can you try trigger this action https://github.com/subhamkrai/rook/actions/workflows/create-tag.yaml here